pkg/utils/map_utils.go

```diff
@@ -33,3 +33,14 @@ func SortMapByKeysAndValuesUniq(m map[string][]string) map[string][]string {
 	}
 	return res
 }
+
+func MapOfInterfacesToMapOfStrings(input map[any]any) map[string]any {
+	converted := make(map[string]any)
+	for key, value := range input {
+		strKey, ok := key.(string)
+		if ok {
+			converted[strKey] = value
+		}
+	}
+	return converted
+}
```

Comment:
Consider preallocating the slice to the proper length to prevent reallocation during the function call...

```
func MapOfInterfacesToMapOfStrings(input map[any]any) map[string]any {
	converted := make(map[string]any, len(input)) // Preallocate map with the same size
	for key, value := range input {
		if strKey, ok := key.(string); ok {
			converted[strKey] = value
		}
	}
	return converted
}
```